alphabet-Order-task.js

```diff
@@ -0,0 +1,11 @@
+// Napisz funkcję alphabetOrder(), która zwróci podany łańcuch znaków z literami
+// uporządkowanymi według alfabetu („wyraz” - „arwyz”). Nie przejmuj się na razie polskimi
+// znakami :)
+
+function alphabetOrder(sortingAscending) {
```

Comment:
This is not a good name for this parameter. It's not sorted. It's a regular string. Every possible string.